packages/cip2/src/RoundRobinRandomImprove/change.ts

```diff
@@ -234,9 +262,31 @@ const computeChangeBundles = (
   }
   return { changeBundles, ...utxoSelection };
 };
+
+const changeBundlesToValues = (
+  csl: CardanoSerializationLib,
+  changeBundles: ValueQuantities[],
+  tokenBundleSizeExceedsLimit: TokenBundleSizeExceedsLimit
+) => {
+  const values = changeBundles.map((bundle) => valueQuantitiesToValue(bundle, csl));
```

Comment:
I think renaming this so we know it' explicit behaviour prior to the removal when the Ogmios type is updated. 
```suggestion
  const values = changeBundles.map((bundle) => ogmiosValueToCslValue(bundle, csl));
```